### PR TITLE
feat: redefine object fields for arana-UI

### PR DESCRIPTION
**Pull request by GavinLam164**
What this PR does:

redefine object fields for arana-UI

Which issue(s) this PR fixes:

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

### CODE REVIEW

File: pkg/admin/router/clusters.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package router
 
 import (
 	"context"
+	"github.com/arana-db/arana/pkg/config"
```
Comment (dongzl) [inline pkg/admin/router/clusters.go:22]: please format import, use this tool:

```
go get -u github.com/dubbogo/tools/cmd/imports-formatter

imports-formatter
```

File: pkg/admin/router/db_groups.go [hunk 2]
```diff
@@ -19,6 +19,7 @@ package router
 
 import (
 	"context"
+	"github.com/arana-db/arana/pkg/config"
```
Comment (dongzl) [inline pkg/admin/router/db_groups.go:22]: The same.

**Review by dongzl — CHANGES_REQUESTED** [review]

**Comment by dongzl** [comment]
Please fix CI, Thanks.

File: pkg/admin/router/db_groups.go [hunk 3]
```diff
@@ -61,12 +62,22 @@ func CreateGroup(c *gin.Context) error {
...
 	tenantName := c.Param("tenant")
-	cluster := c.Param("cluster")
-	groups, err := service.ListGroups(context.Background(), tenantName, cluster)
+	//cluster := c.Param("cluster")
+	clusters, err := service.ListClusters(context.Background(), tenantName)
 	if err != nil {
 		return err
 	}
-	c.JSON(http.StatusOK, groups)
+	var res[] *config.Group
```
Comment (jjeffcaii) [inline pkg/admin/router/db_groups.go:70]: Should be `var res []*config.Group`

File: pkg/boot/proto.go [hunk 4]
```diff
@@ -28,6 +28,7 @@ import (
 
 type Cluster struct {
 	Tenant string                `yaml:"tenant" json:"tenant"`
+	Name string                `yaml:"name" json:"name"`
```
Comment (jjeffcaii) [inline pkg/boot/proto.go:31]: format code

**Review by jjeffcaii — COMMENTED** [review]
pls fix

**Review by jjeffcaii — APPROVED** [review]

**Review by dongzl — APPROVED** [review]

**Review by Lvnszn — APPROVED** [review]

File: pkg/boot/proto.go [hunk 5]
```diff
@@ -28,6 +28,7 @@ import (
 
 type Cluster struct {
 	Tenant string                `yaml:"tenant" json:"tenant"`
+	Name   string                `yaml:"Name" json:"name"`
```
Comment (Lvnszn) [inline pkg/boot/proto.go:31]: Name -> name

**Pull request merged**